[2024-10-17T04:09:57.338+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T04:09:57.466+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T04:09:57.523+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T04:09:57.531+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-17T04:09:57.610+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data_purchases_directory> on 2024-10-16 00:00:00+00:00
[2024-10-17T04:09:57.669+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=507) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-17T04:09:57.716+0000] {standard_task_runner.py:72} INFO - Started process 525 to run task
[2024-10-17T04:09:57.735+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'extract_data_purchases_directory', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmp50mi69d7']
[2024-10-17T04:09:57.765+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask extract_data_purchases_directory
[2024-10-17T04:09:58.530+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [running]> on host 17ff951a544b
[2024-10-17T04:09:58.948+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='extract_data_purchases_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T04:09:58.955+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T04:09:59.008+0000] {logging_mixin.py:190} INFO - Directory  base_datos/compras not found.
[2024-10-17T04:09:59.010+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-17T04:09:59.063+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T04:09:59.064+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_update_stock_ferrimac, task_id=extract_data_purchases_directory, run_id=scheduled__2024-10-16T00:00:00+00:00, execution_date=20241016T000000, start_date=20241017T040957, end_date=20241017T040959
[2024-10-17T04:09:59.138+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-17T04:09:59.199+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-17T04:09:59.204+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-10-17T15:08:08.441+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T15:08:08.737+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T15:08:08.813+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T15:08:08.842+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-17T15:08:09.018+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data_purchases_directory> on 2024-10-16 00:00:00+00:00
[2024-10-17T15:08:09.076+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=168) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-17T15:08:09.112+0000] {standard_task_runner.py:72} INFO - Started process 193 to run task
[2024-10-17T15:08:09.119+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'extract_data_purchases_directory', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmplyu36cmz']
[2024-10-17T15:08:09.151+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask extract_data_purchases_directory
[2024-10-17T15:08:09.489+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [running]> on host df4e8ab00e1c
[2024-10-17T15:08:09.876+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='extract_data_purchases_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T15:08:09.886+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T15:08:09.962+0000] {logging_mixin.py:190} INFO - Directory  base_datos/compras not found.
[2024-10-17T15:08:09.963+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-17T15:08:10.022+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T15:08:10.035+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_update_stock_ferrimac, task_id=extract_data_purchases_directory, run_id=scheduled__2024-10-16T00:00:00+00:00, execution_date=20241016T000000, start_date=20241017T150808, end_date=20241017T150810
[2024-10-17T15:08:10.130+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-17T15:08:10.227+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-17T15:08:10.237+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-10-17T16:07:17.080+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T16:07:17.188+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T16:07:17.239+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T16:07:17.240+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-17T16:07:17.306+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data_purchases_directory> on 2024-10-16 00:00:00+00:00
[2024-10-17T16:07:17.350+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=102) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-17T16:07:17.366+0000] {standard_task_runner.py:72} INFO - Started process 118 to run task
[2024-10-17T16:07:17.424+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'extract_data_purchases_directory', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmp3igf4vgl']
[2024-10-17T16:07:17.470+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask extract_data_purchases_directory
[2024-10-17T16:07:18.039+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [running]> on host 29817b8f615d
[2024-10-17T16:07:19.321+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='extract_data_purchases_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T16:07:19.326+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T16:07:19.419+0000] {logging_mixin.py:190} INFO - Directory  base_datos/compras not found.
[2024-10-17T16:07:19.426+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-17T16:07:19.538+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T16:07:19.540+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_update_stock_ferrimac, task_id=extract_data_purchases_directory, run_id=scheduled__2024-10-16T00:00:00+00:00, execution_date=20241016T000000, start_date=20241017T160717, end_date=20241017T160719
[2024-10-17T16:07:19.669+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-17T16:07:19.826+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-17T16:07:19.852+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-10-17T16:14:28.711+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T16:14:28.770+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T16:14:28.811+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T16:14:28.812+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-17T16:14:28.909+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data_purchases_directory> on 2024-10-16 00:00:00+00:00
[2024-10-17T16:14:28.930+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=82) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-17T16:14:28.934+0000] {standard_task_runner.py:72} INFO - Started process 98 to run task
[2024-10-17T16:14:28.937+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'extract_data_purchases_directory', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmpw4kjvg1v']
[2024-10-17T16:14:28.946+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask extract_data_purchases_directory
[2024-10-17T16:14:29.171+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [running]> on host 1ab1bf654054
[2024-10-17T16:14:29.681+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='extract_data_purchases_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T16:14:29.685+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T16:14:29.761+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/functions_etl/extract_file_purchases_to_stock.py", line 24, in extract_file_purchases_to_stock
    archivo_origen = origin_path.joinpath(compras_file)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/pathlib.py", line 717, in joinpath
    return self.with_segments(self, *pathsegments)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/pathlib.py", line 385, in with_segments
    return type(self)(*pathsegments)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/pathlib.py", line 1162, in __init__
    super().__init__(*args)
  File "/usr/local/lib/python3.12/pathlib.py", line 373, in __init__
    raise TypeError(
TypeError: argument should be a str or an os.PathLike object where __fspath__ returns a str, not 'NoneType'
[2024-10-17T16:14:29.825+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=etl_update_stock_ferrimac, task_id=extract_data_purchases_directory, run_id=scheduled__2024-10-16T00:00:00+00:00, execution_date=20241016T000000, start_date=20241017T161428, end_date=20241017T161429
[2024-10-17T16:14:29.875+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T16:14:29.878+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 7 for task extract_data_purchases_directory (argument should be a str or an os.PathLike object where __fspath__ returns a str, not 'NoneType'; 98)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/functions_etl/extract_file_purchases_to_stock.py", line 24, in extract_file_purchases_to_stock
    archivo_origen = origin_path.joinpath(compras_file)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/pathlib.py", line 717, in joinpath
    return self.with_segments(self, *pathsegments)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/pathlib.py", line 385, in with_segments
    return type(self)(*pathsegments)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/pathlib.py", line 1162, in __init__
    super().__init__(*args)
  File "/usr/local/lib/python3.12/pathlib.py", line 373, in __init__
    raise TypeError(
TypeError: argument should be a str or an os.PathLike object where __fspath__ returns a str, not 'NoneType'
[2024-10-17T16:14:29.948+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-17T16:14:30.086+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-17T16:14:30.092+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-10-17T17:20:11.606+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T17:20:11.801+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T17:20:12.018+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T17:20:12.076+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-17T17:20:12.212+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data_purchases_directory> on 2024-10-16 00:00:00+00:00
[2024-10-17T17:20:12.265+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=107) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-17T17:20:12.433+0000] {standard_task_runner.py:72} INFO - Started process 132 to run task
[2024-10-17T17:20:12.452+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'extract_data_purchases_directory', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmp89zwjz0q']
[2024-10-17T17:20:12.505+0000] {standard_task_runner.py:105} INFO - Job 6: Subtask extract_data_purchases_directory
[2024-10-17T17:20:13.387+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [running]> on host 1652cc614667
[2024-10-17T17:20:15.714+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='extract_data_purchases_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T17:20:15.732+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T17:20:16.556+0000] {logging_mixin.py:190} INFO - File 'compras_stock.xlsx' moved to stock folder correctly.
[2024-10-17T17:20:16.558+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-17T17:20:16.604+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T17:20:16.606+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_update_stock_ferrimac, task_id=extract_data_purchases_directory, run_id=scheduled__2024-10-16T00:00:00+00:00, execution_date=20241016T000000, start_date=20241017T172011, end_date=20241017T172016
[2024-10-17T17:20:16.733+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-17T17:20:16.867+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-17T17:20:16.874+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-10-17T22:09:00.906+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T22:09:00.964+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T22:09:00.991+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T22:09:00.993+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-17T22:09:01.030+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data_purchases_directory> on 2024-10-16 00:00:00+00:00
[2024-10-17T22:09:01.046+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=145) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-17T22:09:01.056+0000] {standard_task_runner.py:72} INFO - Started process 154 to run task
[2024-10-17T22:09:01.052+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'extract_data_purchases_directory', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmpcxm36cyh']
[2024-10-17T22:09:01.061+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask extract_data_purchases_directory
[2024-10-17T22:09:01.243+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.extract_data_purchases_directory scheduled__2024-10-16T00:00:00+00:00 [running]> on host d95e36bcd28a
[2024-10-17T22:09:01.812+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='extract_data_purchases_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T22:09:01.839+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T22:09:02.250+0000] {logging_mixin.py:190} INFO - File 'compras_stock.xlsx' moved to stock folder correctly.
[2024-10-17T22:09:02.252+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-17T22:09:02.303+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T22:09:02.304+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_update_stock_ferrimac, task_id=extract_data_purchases_directory, run_id=scheduled__2024-10-16T00:00:00+00:00, execution_date=20241016T000000, start_date=20241017T220900, end_date=20241017T220902
[2024-10-17T22:09:02.523+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-17T22:09:02.591+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
